### ISSUE TITLE
Set default value for managed metadata column

### ISSUE THREAD

**Issue by JamesSH-bytes** [pnp/pnpframework#87]
what would be the best way to set the default value for managed metadata columns? I was trying with powershell using this: 
Set-PnPField -Identity "ProductCategory" -Values @{DefaultValue = "5;#Laptops|b25434dc-fcf3-3a43-c43a-d4aa1234a2cc"}

but the value I got from the provisioning template is not always this: 5;#Laptops|b25434dc-fcf3-3a43-c43a-d4aa1234a2cc as the number 5;# changes from tenants. is there a way to do this with xml in the template or with powershell?

**Comment by JamesSH-bytes** [comment]
@YannickRe @jansenbe do you have an idea on doing this?

**Comment by YannickRe** [comment]
That's because the default value is based on the Taxonomy Hidden List, and your "5" is essentially the List Item ID of the term in the Taxonomy Hidden List. The number is variable.
I haven't tried for default value nor through PnP PowerShell,  but try "-1" instead of "5". I seem to remember from my Server Side Dev days that this was the workaround we used 😅

**Comment by jansenbe** [comment]
@JamesSH-bytes : are you unblocked, can we close this issue?

**Comment by jansenbe** [comment]
Closing this one per no additional feedback. Please create a new issue taken in account above comments if the problem still persists
